src/gui/folderman.cpp

```diff
@@ -1355,9 +1355,12 @@ static QString checkPathValidityRecursive(const QString &path)
     Utility::NtfsPermissionLookupRAII ntfs_perm;
 #endif
     const QFileInfo selFile(path);
+    if (!selFile.dir().entryList({ QStringLiteral(".sync_*.db") }, QDir::Hidden | QDir::Files).isEmpty()) {
```

Comment:
Hmm I'd prefer to keep * with the change to .sync_journal.db .
Maybe one day we want the branding name in there or something similar and the check would fail for no reason.